backend/app/services/database.py

```
@@ -797,3 +769,58 @@ def track_script_edit(self, script_id: str, user_id: str, action_type: str, meta
+                # Removed 'message' column to match schema.sql
+                conn.execute(
+                    '''
+                    INSERT INTO versions (id, script_id, content, created_by)
+                    VALUES (?, ?, ?, ?)
+                    ''',
+                    (str(uuid.uuid4()), script_id, content, user_id)
+                )
+
+                return script_id
+        except Exception as e:
+            logger.error(f"Failed to create script: {str(e)}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)